pkg/jx/cmd/create_project.go

```diff
@@ -0,0 +1,158 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/jenkins-x/jx/pkg/jx/cmd/templates"
+	"github.com/jenkins-x/jx/pkg/util"
+	"github.com/spf13/cobra"
+	"gopkg.in/AlecAivazis/survey.v1/terminal"
+	"io"
+	"os"
+)
+
+const (
+	createQuickstartName = "Create new application from a Quickstart"
+	createCamelName      = "Create new Apache Camel microservice"
+	createSpringName     = "Create new Spring Boot microservice"
+	importDirName        = "Import existing code from a directory"
+	importGitName        = "Import code from a git repository"
+	importGitHubName     = "Import code from a github repository"
```

Comment:
I wondering if these shouldn't be more generic than just GitHub.

Reply:
yeah - we should be able to support importing from any git repository I think; we should probably fix that in `jx import` too